safe_control_gym/envs/benchmark_env.py

```diff
@@ -443,8 +443,8 @@ def before_step(self, action):
 
         self.current_raw_action = action
         # Pre-process/clip the action
-        processed_action = self._preprocess_control(action)
-        return processed_action
+        # processed_action = self._preprocess_control(action)
+        return action
```

Comment:
Yes, this will change the behaviour. The primary necessity was that the attitude controller needs to run at a higher frequency and is made to be a part of the preprocess function. Then, the intent is to run "preprocess_action" at the same frequency as pyb in advance_simulation. 
Corresponding care is taken to correct it in cartpole.

Reply:
I am not sure if y'all discussed this yet but we should talk about the best way to change the preprocess function to accomodate the attitude controller then. I don't understand what the impacts of this change are yet